### DISCUSSION TITLE
webman mod v.1.47.40 issue

### DISCUSSION THREAD

**Discussion by alekslonewolf1992** [aldostools/webMAN-MOD#726] [Q&A] [answered]
***V.1.47.41 ISSUE***

_Hi Aldo, thank you for your work & update !

I wanto to report  an issue in this WM version.
tested on 2 PS3 - first SuperSlim 4004C - second Slim 2504B  - both on hen 3.1.1 & webman 1.47.41 default setup options 

Every time I launch the XML refresh with ntfs hard/usb drive plugged, ps3 freezes always!. Sometimes freezes immediatly after enable hen.
 
I changed hard drive, usb drive...  I tried a lot of attempts... 
curious fact: in one of my attemps... I unchecked "dev_ntfs" in wM setup, I installed & using prepiso1.30 to update manually games list in webman . No freeze happened anymore. Then I checked again "dev_ntfs" , with or without prepiso installed.. and the freezes reappeared !

Maybe there are some problems in internal webman dev.ntfs scanning?? 

anyway... At the end, my last chance... I reinstalled hfw&hen, unistalled WM v.41, and tested WM .40 version.... and it worked ! I solved with a downgrade of webman.

so... Can you make an attempt to recreate this freeze?

I'm not the only one with problems about external drives & refresh with this latest version..
I hope i was helpful... 
Thank you again for the great work and have a nice day !_

**Comment by aldostools** [comment, marked as answer]
This issue was answered here:
https://github.com/aldostools/webMAN-MOD/issues/727

There is not a solution, due the internal ntfs library used by wMM is very limited.
